### ISSUE TITLE
imminent deprecation of airport command

### ISSUE THREAD

**Issue by ofirgalcon** [munkireport/wifi#7]
macOS 14.4 beta says this when trying to run the airport cmd:

`WARNING: The airport command line tool is deprecated and will be removed in a future release. For diagnosing Wi-Fi related issues, use the Wireless Diagnostics app or wdutil command line tool.` 

/usr/libexec/airportd also doesn't work

```DESCRIPTION
     wdutil provides functionality of the Wireless Diagnostics application in command line form.

COMMANDS
     The following commands are available:

     diagnose [-f outputDirectoryPath]

           Equivalent to running the Wireless Diagnostics application, without UI. The default
           outputDirectoryPath is /var/tmp.

           Requires sudo.

     info

           Displays all of the content included in the Wi-Fi Info utility view.

     log [{+|-} {dhcp|od|dns|eapol|wifi}]+

           Enables or disables logging for DHCP, OpenDirectory, DNS, EAPOL, and Wi-Fi

           Requires sudo.

     dump

           Dumps the temporary Wi-Fi log buffer to /tmp/wifi-XXXXXX.log

EXAMPLES
     Run all diagnostic tests:

           sudo wdutil diagnose

     Gather information about the current wireless environment:

           wdutil info

     Enable DHCP and OpenDirectory logging:

           sudo wdutil log +dhcp +od

     Disable EAPOL logging:

           sudo wdutil log -eapol

AUTHOR
     This program and document are maintained by Apple Inc. <wifi-diags@group.apple.com>.```

**Comment by precursorca** [comment]
Examining the newer sudo wdutil info here are some equivalents

airport -I -> sudo wdutil info

agrCtlRSSI ->  RSSI
agrExtRSSI -> 
agrCtlNoise -> Noise
agrExtNoise ->
state -> Power
op mode -> Op Mode
lastTxRate -> Tx Rate
maxRate -> 
lastAssocStatus -> 
802.11 auth -> 
link auth -> WPA2 Personal
BSSID -> BSSID
SSID ->  SSID
MCS ->  MCS Index 
guardInterval -> Guard Interval
NSS ->  NSS
channel -> Channel

**Comment by tuxudo** [comment]
This has been resolved with version v3.5 of the module
